octopoes/.ci/docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       args:
         ENVIRONMENT: dev
       context: .
-    command: pytest tests/integration --timeout=300
+    command: pytest -s tests/integration/test_internetnl_nibble.py --timeout=300
```

Comment:
probably not what you wanted to commit?

Reply:
I'm leaving it till the draft is completed.

Reply:
Context: these are branches we pass back and forth a lot so we want to directly in the test we are working on